src/Collapse/Collapse.component.tsx

```diff
@@ -21,36 +22,124 @@ interface CollapseProps {
     openedIcon?: React.ReactElement;
     /** Icon to show for the closed state of the component */
     closedIcon?: React.ReactElement;
-    /** 2 possible themes, compact (doc site) and comfortable (RMN CBO), and also none to allow for styling */
-    theme?: Themes;
+    /** 2 possible variants, compact (doc site) and comfortable (RMN CBO), and also none to allow for styling */
+    variant?: CollapseVariants;
     /** This prop is solely used by the CollapseGroup component to make Collapse components have accordion behavior */
-    accordionHandler?: any;
+    onClick?: any;
     /** This mostly exists for CollapseGroup in order to hide the bottom border of stacked Collapse children */
     hasBottomBorder?: boolean;
     children?: any;
     className?: string;
 }
 
-interface StyledCollapseProps {
-    theme?: string;
+interface StyledCollapseProps extends CollapseProps {
+    variant?: string;
     hasBottomBorder?: boolean;
 }
 
 export const DEFAULT_OPENED_TEXT = 'Close';
 export const DEFAULT_CLOSED_TEXT = 'Open';
-const DEFAULT_THEME = 'comfortable';
+const DEFAULT_VARIANT = 'comfortable';
 
-// Associates files from Collapse/Themes directory to specific theme names
-export const CollapseThemes: { [key: string]: FlattenSimpleInterpolation } = {
-    comfortable: ComfortableTheme,
-    compact: CompactTheme,
-    none: css``,
+const variants = {
+    comfortable: css`
+        padding: 1rem 2rem;
+        border-top: solid ${colors.silver.dark} 1px;
```

Comment:
I think this should be:
```border-top: solid thin ${th.color('borders.dark')};```
But, I don't think you had all the colors defined when this PR was submitted. I've updated the root branch to have all that stuff. How should we attack updating this PR? Want me to handle that?